src/mozanalysis/metrics/fenix.py

```diff
@@ -0,0 +1,95 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from pyspark.sql import functions as F
+
+from mozanalysis.metrics import Metric, DataSource, agg_sum
+from mozanalysis.utils import all_
+from mozanalysis.udf import extract_search_counts
+
+
+@DataSource.from_func()
+def fenix_baseline(spark, experiment):
```

Comment:
Let's call this `baseline` since it's in the `fenix` module? Similarly for `fenix_metrics` and `fenix_events` below.